testng-core/src/test/java/test/configuration/sample/ExternalConfigurationClass.java

```diff
@@ -1,4 +1,4 @@
-package test.configuration;
+package test.configuration.sample;
```

Comment:
By convention, suffix the class name by `Sample` too

Reply:
done